### PR TITLE
Use 'curl' with '--retry' instead of plain 'wget' in Makefiles

### PR DESCRIPTION
**Pull request by coutinhop**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: node/Makefile [hunk 1]
```diff
@@ -511,14 +511,14 @@ $(WINDOWS_ARCHIVE_ROOT)/confd/templates/%: windows-packaging/templates/%
...
+	curl -sSf -L --retry 5 -o $@ $(MICROSOFT_SDN_GITHUB_RAW_URL)/Kubernetes/windows/hns.psm1
 
 $(WINDOWS_ARCHIVE_ROOT)/libs/hns/License.txt:
-	wget -P $(WINDOWS_ARCHIVE_ROOT)/libs/hns/ $(MICROSOFT_SDN_GITHUB_RAW_URL)/License.txt
+	curl -sSf -L --retry 5 -o $@  $(MICROSOFT_SDN_GITHUB_RAW_URL)/License.txt
 
 ## Download NSSM.
 windows-packaging/nssm.zip:
-	wget -O windows-packaging/nssm.zip https://nssm.cc/ci/nssm-$(WINDOWS_NSSM_VERSION).zip
+	curl -sSf -L --retry 5 -o $@ https://nssm.cc/ci/nssm-$(WINDOWS_NSSM_VERSION)-aaa.zip
```
Comment (hjiawei) [inline node/Makefile:521]: Remove `-aaa`?

**Comment by hjiawei** [comment]
NSSM [download site](https://nssm.cc/download) still gives 502 as of now. Do you know if we have alternative mirrors to retry when the main site is down? Alternatively, can we save the necessary utility into our building container (if possible)?

File: node/Makefile (hunk 1 above)
Comment (coutinhop) [reply node/Makefile:521]: Thanks for catching this! I had changed it to manually test the retrying and forgot to clean it up...

**Comment by coutinhop** [comment]
> NSSM [download site](https://nssm.cc/download) still gives 502 as of now. Do you know if we have alternative mirrors to retry when the main site is down? Alternatively, can we save the necessary utility into our building container (if possible)?

The errors are happening transiently (I can now access the nssm download page fine). We could look into adding the nssm zip (or executable) to our repo, I guess (not sure if that has any license implications). I don't know of any mirrors we could use, unfortunately...

**Review by lwr20 — APPROVED** [review]

**Pull request merged**